modular_skyrat/modules/customization/modules/clothing/hands/hands.dm

```diff
@@ -31,7 +31,7 @@
 	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
 	resistance_flags = NONE
 
-/obj/item/clothing/gloves/bracer/wraps
+/obj/item/clothing/gloves/wraps  //BUBBER EDIT (removed "bracer" path before "wraps")
```

Comment:
Make your own type, or override the armor values, removing the skyrat one with no overrides might result in invalid type path issues whenever they choose to add these elsewhere in the code